Forwarded: https://github.com/mariadb-corporation/libmarias3/pull/102
Forwarded: https://github.com/facebook/rocksdb/pull/7785
Author: Otto Kekäläinen <otto@debian.org>
Subject: Fix various spelling errors still found in code
  Two upstream PRs remain. They have been merged, but not imported on MariaDB 10.5.

--- a/storage/maria/libmarias3/src/xml.c
+++ b/storage/maria/libmarias3/src/xml.c
@@ -782,7 +782,7 @@ static struct xml_node* xml_parse_node(s
 	/* Close tag has to match open tag
 	 */
 	if (!xml_string_equals(tag_open, tag_close)) {
-		xml_parser_error(parser, NO_CHARACTER, "xml_parse_node::tag missmatch");
+		xml_parser_error(parser, NO_CHARACTER, "xml_parse_node::tag mismatch");
 		goto exit_failure;
 	}
 
--- a/storage/rocksdb/rocksdb/include/rocksdb/cache.h
+++ b/storage/rocksdb/rocksdb/include/rocksdb/cache.h
@@ -60,7 +60,7 @@ struct LRUCacheOptions {
   // If greater than zero, the LRU list will be split into a high-pri
   // list and a low-pri list. High-pri entries will be insert to the
   // tail of high-pri list, while low-pri entries will be first inserted to
-  // the low-pri list (the midpoint). This is refered to as
+  // the low-pri list (the midpoint). This is referred to as
   // midpoint insertion strategy to make entries never get hit in cache
   // age out faster.
   //
